packages/world/src/systems/BuildSystem.sol

```diff
@@ -224,12 +224,16 @@ library BuildLib {
 
     EntityId base = _addBlock(ctx.buildType, ctx.coord);
     EntityOrientation._set(base, ctx.orientation);
+
     Mass._setMass(base, ObjectPhysics._getMass(ctx.buildType));
```

Comment:
Do we need to set this mass here now if we're doing it inside the `EntityUtils.setEntityObjectType`?

Reply:
Also do we even need the new call here? We weren't calling `EntityObjectType._set` here before for base

Reply:
_set was being called in applyTerrainModifications, which I removed. But yes this _setMass is unnecessary as it is already done by setEntityObjectType